101-machine-learning-create/azuredeploy.json

```diff
@@ -56,7 +56,7 @@
       },
       "storageAccountName": {
         "type": "string",
-        "defaultValue": "[concat('sa',uniqueString(resourceGroup().id))]"
+        "defaultValue": "[concat('sa',uniqueString(resourceGroup().id, deployment().name))]"
```

Comment:
these won't be idempotent - what's the scenario you ran into for needing a change here?

Reply:
In order to ensure the uniqueness when using different deployment names.

Reply:
Without this change, if we deploy the second time(without deployment name parameter), the resource names are the same at sometime.

Reply:
That's the intent - when deploying to the same RG you can redeploy idempotently.  If I want a new instance, I can change the param value, if I don't, this approach makes it difficult to do (and goes against the patterns in the repo.

So let's remove that from the seed

Reply:
Got it.